bliss/models/decoder.py

```diff
@@ -188,7 +190,7 @@ def forward(self):
         return psf
 
     def _get_psf(self):
-        psf = torch.tensor([], device=device)
+        psf = torch.tensor([], device=self.device)
         for i in range(self.n_bands):
             _psf = self._get_psf_single_band(self.params[i])
```

Comment:
hmm do you need to make sure that `psf_params` are in the correct device for this line to work ? (maybe this is why you have the `.to()` outside when you create the decoder?

Reply:
Sorry I'm not sure which one you meant. `psf` should be in the correct device since it's created on `self.device`.
`psf_params` here is automatically managed by pytorch along with other parameters. So it doesn't need manually assign. 
https://github.com/prob-ml/bliss/blob/1802f0aefdc44d0bb482f1575f328954b4ddfe4c/bliss/models/decoder.py#L158-L170